### PR TITLE
Release notes release-2024-09

### PR DESCRIPTION
**Pull request by robinbisping**
- Side panel space optimization (https://github.com/livingdocsIO/livingdocs-editor/pull/8760, https://github.com/livingdocsIO/livingdocs-editor/pull/8803)
- Retresco main entities (https://github.com/livingdocsIO/livingdocs-server/pull/7164)
- iMatrics test environments (https://github.com/livingdocsIO/livingdocs-server/pull/7157)

### CODE REVIEW

File: content/operations/releases/release-2024-09.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ In your .nvmrc (if present) change the string from 18 to 22 or 20.
 
 ### Space optimisation for side panels :gift:
 
-
+The width of wide side panels (e.g., document inbox, media library) has been increased on large screens to create additional space.
```
Comment (JordiVM) [inline content/operations/releases/release-2024-09.md:149]: Do you think having a screenshot would help there?

Comment (JordiVM) [inline content/operations/releases/release-2024-09.md:149]: We want to stress here that the changes shouldn't change the editor for the worst so if users observe any component overlap (within livingdocs editor or their design) they should contact us.

```suggestion
The width of wide side panels (e.g., document inbox, media library) has been increased on large screens to create additional space.

Teaser side panel and Document Inbox will get the extended width to match the Media Library side panel (600px). In wider than 1920px screens, those panels will extend to 800px.

If your users observe there are overlapping components that affect negatively the experience reach out to your CSO manager. 
```

File: content/operations/releases/release-2024-09.md [hunk 2]
```diff
@@ -166,11 +166,17 @@ In your .nvmrc (if present) change the string from 18 to 22 or 20.
...
+Retresco has introduced a new concept of main entities. These are entities identified by Retresco that are particularly relevant to a document (i.e., they have a high score). To support this new concept, we have extended our storage schema for Retresco entities to include an indicator `isMain` of whether an entity is a main entity and also report this data back to Retresco.
 
+No changes are required for main entities to work, other than enabling main entities in Retresco. Currently, main entities are not marked in the Livingdocs UI. They appear as regular entities.
 
 ### iMatrics test environment support :gift:
 
+To support iMatrics test environments, we are adding a new iMatrics project configuration property to specify whether the iMatrics test environment should be used. By default, it is set to `false`.
 
+```js
+testEnvironment: true
```
Comment (JordiVM) [inline content/operations/releases/release-2024-09.md:178]: integrations: {
  imatrics: {
    testEnvironment: true
  }
}

File: content/operations/releases/release-2024-09.md (hunk 1 above)
Comment (robinbisping) [reply content/operations/releases/release-2024-09.md:149]: Thank you, Jordi! Now I get it. I was a bit confused why we included it in the release notes. But it makes sense now.

**Review by JordiVM — APPROVED** [review]

**Pull request merged**